src/cyhy_db/models/place_doc.py

```diff
@@ -0,0 +1,38 @@
+"""The model for place documents."""
+
+# Standard Python Libraries
+from typing import Optional
+
+# Third-Party Libraries
+from beanie import Document
+from pydantic import ConfigDict, Field
+
+
+class PlaceDoc(Document):
+    """The place document model."""
+
+    model_config = ConfigDict(extra="forbid")
+
+    clazz: str = Field(alias="class")  # 'class' is a reserved keyword in Python
```

Comment:
We should be consistent with how we handle these cases (e.g. `class_` -> `class`).  We also need to nail down the "alias" vs. "serialization_alias" business.  For reference: https://github.com/cisagov/cyhy-db/blob/5b2bb40c9984243cc01a870175ff07ad05167925/src/cyhy_db/models/ticket_doc.py#L30-L33

Reply:
I think `clazz` is a pretty common alternate name.  It's used a lot in Java, e.g., where the need to name things `class` comes up surprisingly often.

Also, I like to think of `clazz` as being a classy guy _but with an edge_.  Like a James Bond sort of character.

Reply:
I don't have anything against `clazz`, but if `class` is aliased as `clazz`, then should `from` be aliased as `fromm`, `fromz`, `frozz`, or something else?

It just seems like appending an underscore to create the alias is a simpler way to go:
`class` -> `class_`
`from` -> `from_`

Reply:
This most certainly was my Java neurons solving a reserved word problem.  I agree that it should be normalized with our previous usage of `from_`.  Although `fromz` is tempting.

Reply:
Resolved in:
- 07724b23d80129fea518e373f5825469882bc800

Reply:
Did you also confirm if we want `alias` or `serialization_alias`, [as mentioned above](https://github.com/cisagov/cyhy-db/pull/5#discussion_r1801711616)?

Reply:
I think I've tracked it down.  `alias` should be all we need but I see evidence of either a bug, a lack of documentation, or my own misunderstanding of how all this works.

I think it has to do with how aliases are being applied to sub-documents.  This leads me to believe that it is a bug: 
- https://github.com/BeanieODM/beanie/issues/369

But the `populate_by_name` model configuration option of Pydantic leads me to believe it might be by design...
- https://docs.pydantic.dev/latest/api/config/#pydantic.config.ConfigDict.populate_by_name

In any case, if we just set up our alias like this it fails with a Pydantic validation error:
```python
    from_: Optional[bool | float | int | str] = Field(..., alias="from")
```

But if I set `populate_by_name` to `true` it works as expected: 
```python
class EventDelta(BaseModel):
    """The event delta model."""

    model_config = ConfigDict(populate_by_name=True)

    from_: Optional[bool | float | int | str] = Field(..., alias="from")
```

So `populate_by_name` needs to be `true`, as either the proper way to do this, or as a work-around to a bug.